identity/app/services/PlaySigninService.scala

```diff
@@ -24,4 +24,13 @@ class PlaySigninService(conf: IdentityConfiguration) extends SafeLogging {
       }
     }
   }
+
+  def getCookies(apiCookies: CookiesResponse, rememberMe: Boolean)(implicit executionContext: ExecutionContext): List[Cookie] = {
```

Comment:
Probably worth having a private method that consolidates the copied code here and gets called by both versions of `getCookies`.

Reply:
Actually scratch that - I think you should be able to re-use the existing one? Look at how the Reauthentication controller does it, looks like that should work here.